cmd/helm/flags.go

```diff
@@ -1,5 +1,6 @@
 /*
 Copyright The Helm Authors.
+Copyright (c) 2024 Rakuten Symphony India.
```

Comment:
Can we please remove the change in copyright notice from the PR please

Reply:
Removed the copyright notice from the PR.